jcl/src/java.base/share/classes/openj9/internal/criu/InternalCRIUSupport.java

```diff
@@ -403,6 +404,26 @@ public static String getErrorMessage() {
 	private boolean unprivileged;
 	private Path envFile;
 	private String optionsFile;
+	private int ghostFileLimit;
+
+	/**
+	 * Set the size limit for ghost files when taking a checkpoint. File limit
+	 * can not be greater than 2^32.
+	 *
+	 * Default: 1MB
+	 *
+	 * @param fileLimit the file limit size in bytes.
+	 * @return this
+	 * @throws UnsupportedOperationException if file limit is greater than 2^32 or less than zero.
+	 */
+	public InternalCRIUSupport setGhostFileLimit(long limit) {
+		if ((limit > (4L * 1024L * 1024L * 1024L)) || (limit < 0)) {
+			throw new UnsupportedOperationException("Ghost file must be positive and less than 4GB");
```

Comment:
positive -> non-negative?
less than -> not greater than?

Reply:
The guidelines are to use positive logic, makes it simpler to read.